packages/mobile/src/screens/app-screen/AppTabScreen/AppTabScreen.tsx

```diff
@@ -15,35 +19,49 @@ import {
   NotificationUsersScreen
 } from 'app/screens/user-list-screen'
 
-import { useScreenOptions } from './baseStackScreenOptions'
+import { useScreenOptions } from './useTabScreenOptions'
+
+export type AppTabScreenParamList = {
+  Track: { id: ID }
+  Profile: { handle: string }
+  Collection: { id: ID }
+  Favorited: { id: ID; favoriteType: FavoriteType }
+  Reposts: { id: ID; repostType: RepostType }
+  Followers: { userId: ID }
+  Following: { userId: ID }
+  Search: undefined
+  SearchResults: { query: string }
+  NotificationUsers: {
+    id: string // uuid
+    notificationType: NotificationType
+    count: number
+  }
+}
 
 const forFade = ({ current }) => ({
   cardStyle: {
     opacity: current.progress
   }
 })
 
-type BaseNavigatorProps = {
+type AppTabScreenProps = {
   baseScreen: (
     Stack: ReturnType<typeof createStackNavigator>
   ) => React.ReactNode
   Stack: ReturnType<typeof createStackNavigator>
 }
 
 /**
- * This is the base stack that includes common screens
+ * This is the base tab screen that includes common screens
  * like track and profile
  */
-export const BaseStackNavigator = ({
-  baseScreen,
-  Stack
-}: BaseNavigatorProps) => {
+export const AppTabScreen = ({ baseScreen, Stack }: AppTabScreenProps) => {
```

Comment:
I'm a little hesitant on this name because this stack navigator doesn't _necessarily_ need to be in a tab, it just so happens to be. And the root component is a Navigator, so I feel like naming this *Something*Navigator would make more sense?

Reply:
yeah i def here the idea for navigator, but maybe same argument that the screen just so happens to be a navigator? i was also considering putting all the "tab" stuff in the actually FeedScreen, ExploreScreen screens, since it prob makes sense for those screens to compose all the parts of the explore/feed experience?

Reply:
Yeah that makes sense too! I'm good with that

Reply:
word, ill move those over then and we can see how that looks